### DISCUSSION TITLE
sensor NIC capacity

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#2161] [Unsupported Versions] [answered]
Hello,

This is my home lab which I use as a test environment

Host: HP DL320e Gen 8
2 x Intel(R) Xeon(R) CPU E3-1240 V2 @ 3.40GHz
32 GB Ram
4 X 4 TB disks in raid 10
1 X 1 Gb physical NIC for mirror port switch (1 Gb)
1 x 1 Gb physical NIC for mirror port Router (1 Gb)
2 x 1 Gb Network

ESXI 6.7 Build 17167734 
1 x vSwitch VLAN 4095 with physical NIC to mirror port switch
1 x vSwitch VLAN 4095 with physical NIC to mirror port router
1 x vSwitch network

Security Onion 2.3.10 Standalone
8 Cores vCPU’s
16 GB vRam
2.5 TB disk space Thick provisioned
1 x 10 Gb vNIC VMXNET 3 vSwitch network
1 x 10 Gb vNIC VMXNET 3 vSwitch mirror switch
1 x 10 Gb vNIC VMXNET 3 vSwitch mirror Router 

Network
± 16 devices

Now is my quistion a sensor NIC, how much throughput can it have in this setup? 10 Gb or 1 Gb?
I ask this because sometimes I see packet loss in grafana when the throughput gets above 1Gb. The virtual NIC is 10 Gb

Is there a way to see which sensor has packet loss?

![afbeelding](https://user-images.githubusercontent.com/29328171/100781900-f551de80-340b-11eb-97cf-f3ce94aa07ff.png)
![afbeelding](https://user-images.githubusercontent.com/29328171/100781940-04389100-340c-11eb-898e-a8ab9ed77d4d.png)

 Regards

Bart

**Comment by dougburks** [comment, marked as answer]
That second screenshot shows the packet loss is in the PCAP process, so that means that Stenographer dropped some packets as it was doing full packet capture. The most likely scenario is that your disks are mechanical drives and are having trouble writing fast enough to keep up with the traffic. You will most likely want to filter the traffic that Steno is trying to write to disk, starting with port 443 since the overwhelming majority of port 443 traffic should be encrypted and therefore limited value from a pcap perspective:
https://docs.securityonion.net/en/2.3/bpf.html

**Reply by sleepingbel** [reply]
Hello Doug,

Thanks for the explanation, but the real cause was that I logged the same packet twice.
I had configured the mirror port for the switch to send the RX and TX packets, but that is not needed. One of two is enough.

I have never seen this problem with SO 16.04 witch was running for more than 1 year, but we did not have grafana to show us this.
The new SO is great, and the new features are a big improvement for security and technical monitoring

Regards
Bart
